src/wxvx/workflow.py

```diff
@@ -261,19 +338,32 @@ def _meta(c: Config, varname: str) -> VarMeta:
     return VARMETA[c.variables[varname]["name"]]
 
 
-def _statargs(c: Config, varname: str, level: float | None, source: Source) -> Iterator:
+def _statargs(
+    c: Config, varname: str, level: float | None, source: Source, cycle: datetime | None = None
+) -> Iterator:
+    cycles = (
+        Cycles(
+            start=cycle.strftime("%Y-%m-%dT%H:%M:%S"),
+            step="01:00:00",
+            stop=cycle.strftime("%Y-%m-%dT%H:%M:%S"),
+        )
+        if isinstance(cycle, datetime)
+        else c.cycles
+    )
```

Comment:
```suggestion
    cyclestr = cycle.strftime("%Y-%m-%dT%H:%M:%S")
    cycles = Cycles(start=cyclestr, step="01:00:00", stop=cyclestr) if cycle else c.cycles
```
I think... 1. DRY out `start`/`stop` value. 2. `cycle` is either a `datetime` or `None`, so I think we can just say `if cycle`.

Out of curiosity, does the value of `step` even matter? If not, we might want to use something like `00:00:00` to avoid giving the impression that the step is really meaningful here. Looking at the logic in `wxvx.times` I _think_ that should work.